src/index.ts

```diff
@@ -278,7 +266,9 @@ async function connectToPort(): Promise<void> {
     connectButton.disabled = false;
   } catch (e) {
     console.error(e);
-    term.writeln(`<ERROR: ${e.message}>`);
+    if (e instanceof Error) {
```

Comment:
Gotta love TypeScript…

Reply:
This one is on JavaScript for not requiring things that are thrown to be errors.